.github/workflows/stats_merged_prs.yml

```diff
@@ -0,0 +1,25 @@
+name: Merged PR stats
+
+on:
+  pull_request:
+    branches:
+      - main
+      - v*-branch
+    types: [closed]
+jobs:
+  record_merged:
+    if: github.event.pull_request.merged == true && github.repository == 'zephyrproject-rtos/zephyr'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
you can drop this and it'll run a bit faster

Reply:
fixed